### PR TITLE
feat: Frame switching methods

### PR DESCRIPTION
**Pull request by viralpickaxe**
Added frame switching methods to the actions builder

Fixes: #5

### CODE REVIEW

**Comment by jasmeettextnow** [comment]
Hey @viralpickaxe Why did you close this PR. Isnt that switchToIframe still needed ?

**Comment by jasmeettextnow** [comment]
@viralpickaxe Please add reviewers who have write access so they can review and merge this PR.
@sipayRT @DudaGod  Can either of you review and merge this PR. 
Would be great help to get the switchToFrame  method in for testing iFrames.

File: lib/tests-api/actions-builder.js [hunk 1]
```diff
@@ -329,6 +329,22 @@ module.exports = class ActionsBuilder {
 
         return this;
     }
+
+    switchToFrame(frameElement) {
+        this._pushAction(this.switchToFrame, function switchToFrame(browser) {
+            return findElement(frameElement, browser).then(browser._wd.frame);
```
Comment (sipayRT) [inline lib/tests-api/actions-builder.js]: you can't use private key `_wd` here - you need to expose `frame` from [here](https://github.com/gemini-testing/gemini/blob/master/lib/browser/new-browser.js#L37)

File: lib/tests-api/actions-builder.js [hunk 2]
```diff
@@ -329,6 +329,22 @@ module.exports = class ActionsBuilder {
...
+        });
+
+        return this;
+    }
+
+    switchToWindow() {
+        this._pushAction(this.switchToWindow, function switchToWindow(browser) {
+            return browser._wd.window('');
```
Comment (sipayRT) [inline lib/tests-api/actions-builder.js]: see comment above

**Pull request closed without merging**